src/main/java/io/kokuwa/maven/k3s/mojo/StartMojo.java

```diff
@@ -66,25 +66,31 @@ public void execute() throws MojoExecutionException {
 			return;
 		}
 
-		startContainer(createContainer());
-	}
-
-	private String createContainer() throws MojoExecutionException {
+		var containerId = dockerUtil().getContainerId().orElse(null);
```

Comment:
Feels a little bit unclear to me. 
I would suggest to sort this code a little bit, to make the status more clear:

	
```
		Optional<String> containerId = dockerUtil().getContainerId();
		if (containerId.isPresent()) {
			if (!Files.exists(getKubeConfig())) {
				dockerClient().stopContainerCmd(containerId.get()).exec();
				getLog().info("Container with id '" + containerId.get() + "' stopped, mount was deleted");
			} else {
				getLog().debug("Container with id '" + containerId.get() + "' found");
			}
		} else {
			containerId = Optional.of(createContainer());
		}
		
		if (containerId.isPresent()) {
			var running = dockerUtil().isRunning(containerId.get());
			if (running) {
				getLog().debug("Container with id '" + containerId.get() + "' is running");
			} else {
				startContainer(containerId.get());
				getLog().debug("Container with id '" + containerId.get() + "' started.");
			}
			
			copyKubeConfigToMountedWorkingDirectory(containerId.get());
			awaitK3sNodesAndPodsReady();	
		}
	}
```